src/integrationTest/java/uk/gov/hmcts/darts/audio/controller/AudioControllerPostAdminApproveMediaMarkedForDeletionIntTest.java

```diff
@@ -127,6 +128,16 @@ void postAdminApproveMediaMarkedForDeletionShouldReturnSuccess() throws Exceptio
                 criteriaBuilder.equal(root.get(AuditEntity_.auditActivity).get("id"), AuditActivity.MANUAL_DELETION.getId())
             ));
 
+        // assert additional audit data
+        assertFalse(caseExpiredAuditEntries.isEmpty());
```

Comment:
May want a .size() check to ensure you get the correct number of results returned